creations/espressif.md

```diff
@@ -3,10 +3,16 @@ Community Allocated Creation IDs for Espressif boards
 
 ## `0x0032_xxxx` - ESP32 dev boards
 *  `0x0032_0001` ESP-EYE
+*  `0x0032_A000` ESP32-LyraT
+*  `0x0032_C000` ESP32-DevKitC
+*  `0x0032_C003` ESP32-DevKitC-VE
 
 ## `0x0052_xxxx` - S2 dev boards
+*  `0x0052_C000` ESP32-S2-DevKitC
 
 ## `0x0053_xxxx` - S3 dev boards
+*  `0x0053_C000` ESP32-S3-DevKitC
+*  `0x0053_0001` ESP32-S3-DevKitM
```

Comment:
These should be able to use the USB PIDs that espressif uses. CircuitPython will use them automatically because Creation IDs are a superset of USB IDs. https://github.com/espressif/usb-pids/blob/main/allocated-pids-espressif-devboards.txt

Reply:
Gotcha. Fixed